{
  "uid" : "4460c85a96f954c7",
  "name" : "Pinterest block",
  "fullName" : "Free blocks.Pinterest block",
  "historyId" : "cb61587e7466ec3d1c2995268abd91a7",
  "time" : {
    "start" : 1595622313100,
    "stop" : 1595622347127,
    "duration" : 34027
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "failed",
  "statusDetails" : {
    "message" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                ",
    "trace" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                \n                    at checkExecSyncError (child_process.js:630:11)\n                    at execSync (child_process.js:666:15)\n                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)\n                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.<computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7",
    "flaky" : false
  },
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusDetails" : {
      "message" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                ",
      "trace" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                \n                    at checkExecSyncError (child_process.js:630:11)\n                    at execSync (child_process.js:666:15)\n                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)\n                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.<computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7",
      "flaky" : false
    },
    "steps" : [ {
      "name" : "Can visit the block editor and add a Pinterest block",
      "time" : {
        "start" : 1595622313102,
        "stop" : 0,
        "duration" : -1595622313102
      },
      "status" : "unknown",
      "statusDetails" : {
        "flaky" : false
      },
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "1c1f2dd2178100b2",
        "name" : "Test failed: Free blocks :: Pinterest block",
        "source" : "1c1f2dd2178100b2.png",
        "type" : "image/png",
        "size" : 44369
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Free blocks"
  }, {
    "name" : "testClass",
    "value" : "Free blocks"
  }, {
    "name" : "testMethod",
    "value" : "Pinterest block"
  }, {
    "name" : "package",
    "value" : "Free blocks"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "extra" : {
    "tags" : [ ]
  },
  "source" : "4460c85a96f954c7.json",
  "statusDetailsSafe" : {
    "message" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                ",
    "trace" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                \n                    at checkExecSyncError (child_process.js:630:11)\n                    at execSync (child_process.js:666:15)\n                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)\n                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.<computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7",
    "flaky" : false
  },
  "statusMessage" : {
    "empty" : false,
    "present" : true
  },
  "parameterValues" : [ ]
}